geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/Services.java

```diff
@@ -327,12 +342,14 @@ public void memberSuspected(ID initiator,
     return this.manager;
   }
 
-  public Locator<ID> getLocator() {
-    return this.locator;
+  public void setLocators(final Locator<ID> locator,
+      final MembershipLocator<ID> membershipLocator) {
+    this.locator = locator;
+    this.membershipLocator = membershipLocator;
```

Comment:
what was a `setLocator()` method is now a `setLocators()` one. do you hate me for this?

by way of justification, these two objects are 1-1 in the design. I think it's a Good Thing™ that there are two (smaller) interfaces instead of one (bigger one)

Reply:
William I like the positively pre-emptive persuasion here... at first glance I was certainly questioning this methods signature...